gen/quick_test.go

```diff
@@ -392,6 +398,11 @@ func TestQuickSuite(t *testing.T) {
 		{Sample: tz.StringMap{}, NoLog: true, Kind: thriftTypedef},
 		{Sample: tz.Primitives{}, NoLog: true, Kind: thriftTypedef},
 		{Sample: tz.StringList{}, NoLog: true, Kind: thriftTypedef},
+		{Sample: tss.StringList{}, NoLog: true, Kind: thriftTypedef},
+		{Sample: tss.FooList{}, NoLog: true, Kind: thriftTypedef},
+		{Sample: tss.MyStringList{}, NoLog: true, Kind: thriftTypedef},
+		{Sample: tss.AnotherStringList{}, NoLog: true, Kind: thriftTypedef},
+		{Sample: tss.StringListList{}, NoLog: true, Kind: thriftTypedef},
```

Comment:
I suspect you don't need the NoLog entries. Those are needed for typedefs of primitives only.

(The `tz.*` types have NoLog because they are testing the zap opt-out flag.)